docker/dockerfiles/Dockerfile.server

```diff
@@ -50,6 +50,7 @@ COPY monitor.py monitor.py
 COPY scripts scripts
 COPY setup.py setup.py
 COPY test_cli.py test_cli.py
+COPY sample_worksheet.py sample_worksheet.py
```

Comment:
I'd move this to `scripts` since it's not an essential script to avoid cluttering the top-level, and name it `create-sample-worksheet.py` for consistency.